cp-connector/pkg/controlplane/controlplane.go

```diff
@@ -82,13 +83,15 @@ func (cp *ControlPlane) Register(ctx context.Context, integration Integration) e
 	cp.logger.Debugf("Registered with integration ID %s", cp.integrationID)
 	registrationData.ID = cp.integrationID
 
+	wg := &sync.WaitGroup{}
+	wg.Add(2)
```

Comment:
I would add a small comment to say one is for eventsource and the other is for subscription source